service/matching/backlog_manager_test.go

```diff
@@ -154,3 +163,146 @@ func TestTaskWriterShutdown(t *testing.T) {
 	err = tlm.backlogMgr.SpoolTask(&persistencespb.TaskInfo{})
 	require.Error(t, err)
 }
+
+func TestApproximateBacklogCountIncrement_taskWriterLoop(t *testing.T) {
+	controller := gomock.NewController(t)
+	defer controller.Finish()
+
+	backlogMgr := newBacklogMgr(controller)
+
+	// Adding tasks on the taskWriters channel
```

Comment:
nit: This is not a criticism, but just a pattern that I've noticed you using the "-ing" tense (the gerund present tense I think) a lot in your comments, and I lean more towards using "add" (the imperative form) or "adds" (the indicative form), so I looked up some comment style guides and I think it would be more standard if you dropped the -ing from these comments.\

It's not at all impactful to the code, and kind of subtle so please don't worry about it too much, but in terms of style this is a note I want to give if it's helpful! Concise comments that follow the same style as other comments do improve readability, which is the point of comments.

Some guides that I think explain my thoughts on this / provide some examples. The first one resonates with my reasoning especially:
- https://carlomilanesi.wordpress.com/2012/02/09/grammar-style-for-code-comments/
- https://www.digitalocean.com/community/tutorials/how-to-write-comments-in-go

Reply:
Interesting, thank you for pointing these out! Shall keep this in mind as I ship out stuff here